core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -249,10 +249,12 @@ public String getMountPoint(AlluxioURI uri) throws InvalidPathException {
   }
 
   /**
-   * Returns a copy of the current mount table, the mount table is a map from Alluxio file system
+   * Gets a copy of the current mount table.
+   * <p>
+   * Returns a copy of the current mount table, which is a map from Alluxio file system
```

Comment:
Missing final point.

Reply:
The sentence ends in the following line.